app/controllers/practices_controller.rb

```diff
@@ -334,7 +335,7 @@ def metrics
   end
 
   def practice_name
-    render 'practices/form/instructions'
```

Comment:
🤔 I'm actually not sure this method is being used anymore

Reply:
Yeah, I think we can get rid of this method.